mongodbatlas/clusters.go

```diff
@@ -408,6 +420,51 @@ func (s *ClustersServiceOp) GetProcessArgs(ctx context.Context, groupID, cluster
 	return root, resp, err
 }
 
+// LoadSampleDataset loads the sample dataset into your cluster.
+// See more: https://docs.atlas.mongodb.com/reference/api/cluster/load-dataset/
+func (s *ClustersServiceOp) LoadSampleDataset(ctx context.Context, groupID, clusterName string) (*SampleDatasetLoadJob, *Response, error) {
+	if err := checkClusterNameParam(clusterName); err != nil {
+		return nil, nil, err
+	}
+
+	basePath := fmt.Sprintf(sampleDatasetLoadPath, groupID)
+	escapedEntry := url.PathEscape(clusterName)
+	path := fmt.Sprintf("%s/%s", basePath, escapedEntry)
+
+	req, err := s.Client.NewRequest(ctx, http.MethodPost, path, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	root := new(SampleDatasetLoadJob)
+	resp, err := s.Client.Do(ctx, req, root)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return root, resp, err
+}
+
+// GetSampleDataset gets the Sample Dataset load job
+// See more: https://docs.atlas.mongodb.com/reference/api/cluster/check-dataset-status/
+func (s *ClustersServiceOp) GetSampleDataset(ctx context.Context, groupID, id string) (*SampleDatasetLoadJob, *Response, error) {
```

Comment:
[nit] since you don't get the actual dataset a rename to include the status part would make this method a bit clearer 
```suggestion
func (s *ClustersServiceOp) GetSampleDatasetStatus(ctx context.Context, groupID, id string) (*SampleDatasetLoadJob, *Response, error) {
```